src/handlers/quote.handler.js

```
@@ -0,0 +1,30 @@
+const { firstFiftyQuote, specificQuote } = require("../services/quote.service");
+
+
+const quoteHandler = async (req, res) => {
+  try {
+    const firstFiftyQuoteArray = await firstFiftyQuote();
+    res.json({
+      firstFiftyQuoteArray
+    }).status(200);
+  } catch (error) {
+    console.error(error);
+  }
+}
+
+const specificQuoteHandler = async (req, res) => {
+  try {
+    const specificQuoteObject = await specificQuote(req.params.index);
+    res.json({
+      specificQuoteObject
+    }).status(200);
+  } catch (error) {
+    console.error(error);
+  }
```

Why not return the error back to client with appropriate error message and error code